OmniDB/OmniDB_app/static/OmniDB_app/lib/omnis_legere/lib/omnis-legere.js

```diff
@@ -802,9 +803,8 @@ function createLegere(p_context = {parent: window, self: 'omnisLegere'}, p_optio
       }
 
       // Render the svg with path lines based on the content positions
-      setTimeout(function(){
-        v_legereControl.renderSvg(v_legereControl.planList);
-      }, 150);
+      v_legereControl.renderSvg(v_legereControl.planList);
```

Comment:
wrap it into settimeout as it was before.
 setTimeout(function(){
        v_legereControl.renderSvg(v_legereControl.planList);
      }, 150);